analyzers/src/SonarAnalyzer.CSharp/Rules/MethodParameterUnused.cs

```diff
@@ -236,7 +236,7 @@ private static IImmutableList<IParameterSymbol> GetUnusedParameters(MethodContex
         private static bool IsMethodUsedAsEventHandlerFunctionOrActionInExpression(IMethodSymbol methodSymbol, ExpressionSyntax expression, SemanticModel semanticModel) =>
             !expression.IsKind(SyntaxKind.InvocationExpression)
             && IsStandaloneExpression(expression)
-            && methodSymbol.Equals(semanticModel.GetSymbolInfo(expression).Symbol?.OriginalDefinition);
+            && methodSymbol.Equals(semanticModel?.GetSymbolInfo(expression).Symbol?.OriginalDefinition);
```

Comment:
Let's add `&& semanticModel != null` as 2nd condition for performance reason. We don't need to evaluate `IsStandaloneExpression` in that case